server/events/vcs/github_status_updater.go

```diff
@@ -0,0 +1,50 @@
+package vcs
```

Comment:
If you're creating a new file anyways, can you move this to the new vcs package? I think i explicitly mentioned all new code should go there (server/vcs/..)